bluer/src/device.rs

```diff
@@ -518,6 +519,13 @@ define_properties!(
                 mt
             }),
         );
+
+        /// The battery percentage of the remote device
+        property(
+            BatteryPercentage, u8,
+            dbus: (BATTERY_INTERFACE, "BatteryPercentage", u8, OPTIONAL),
```

Comment:
```suggestion
            dbus: (BATTERY_INTERFACE, "Percentage", u8, OPTIONAL),
```
In the D-Bus API it's still called `Percentage` though.

Reply:
my bad. 
It's fixed now.